### PR TITLE
feature(linkedin): Added Apply with LinkedIn integration

### PR DESCRIPTION
**Pull request by krsween**

### CODE REVIEW

File: src/app/modal/modal.controller.js [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+        this.linkedInData.header += (userProfile.emailAddress || '') + lineBreak;
+        // Location
+        if (userProfile.location && userProfile.location.name) {
+            this.linkedInData.header += (userProfile.location.name || '') + ', ';
+            if (userProfile.location.country) {
+                this.linkedInData.header += (userProfile.location.country.code.toUpperCase() || '') + hardBreak;
+            }
+        }
+        // Work Experience Block
+        this.linkedInData.resume = 'Work Experience:' + lineBreak;
```
Comment (jgodi) [inline src/app/modal/modal.controller.js]: `Work Experience` should be localized.

File: src/app/modal/modal.controller.js [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+                // Start Date
+                if (userProfile.positions.values[0].startDate) {
+                    this.linkedInData.resume += months[userProfile.positions.values[0].startDate.month - 1] + ' ' + userProfile.positions.values[0].startDate.year + ' - ' || '';
+                }
+                // End Date or 'Present'
+                if (userProfile.positions.values[0].endDate) {
+                    this.linkedInData.resume += months[userProfile.positions.values[0].endDate.month - 1] + ' ' + userProfile.positions.values[0].endDate.year || '';
+                } else {
+                    if (userProfile.positions.values[0].isCurrent) {
+                        this.linkedInData.resume += 'Present';
```
Comment (jgodi) [inline src/app/modal/modal.controller.js]: `Present` should be localized.

Comment (krsween) [inline src/app/modal/modal.controller.js]: So should all the dates.

K.

Comment (jgodi) [inline src/app/modal/modal.controller.js]: Good catch!

**Comment by jgodi** [comment]
This PR LGTM. Just the small localization stuff and we are golden.

**Comment by krsween** [comment]
The query strings for localization on the social networks aren't finished
 either.

K.

**Comment by TheReal** [comment]
LGTM. Merging.

**Pull request merged**